src/main/java/com/microsoft/spring/data/gremlin/exception/GremlinEntityInformationException.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.spring.data.gremlin.exception;
+
+public class GremlinEntityInformationException extends IllegalStateException {
```

Comment:
how about wrap exception to specific error? e.g InvalidGremlinEntityType, InvalidIdField etc.

Reply:
OK, get some information from exception name may be more friendly to user side.